app/src/main/java/com/mapbox/vision/examples/DebugView.kt

```diff
@@ -0,0 +1,130 @@
+package com.mapbox.vision.examples
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Handler
+import android.os.Looper
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import com.mapbox.vision.examples.utils.hide
+import com.mapbox.vision.examples.utils.show
+import com.mapbox.vision.mobile.core.models.FrameStatistics
+import com.mapbox.vision.performance.ModelPerformanceConfig
+import kotlinx.android.synthetic.main.view_debug.view.*
+import java.util.concurrent.TimeUnit
+
+class DebugView @JvmOverloads constructor(
```

Comment:
It's not `DebugView`, I think smt like `FpsPerformanceView` sounds better 

```suggestion
class FpsPerformanceView @JvmOverloads constructor(
```

Reply:
it has also session time and calibration, so it's not just `FpsPerformanceView`,  but agree that `DebugView` is not the best name.
maybe `SessionInfoView`